code/lab/lab.cpp

```diff
@@ -1282,7 +1282,7 @@ void labviewer_make_desc_window(Button * /*caller*/)
 void labviewer_make_ship_window(Button * /*caller*/)
 {
 	GUIObject *cbp;
-	TreeItem *ctip, *stip;
+	TreeItem /**ctip, */*stip;
```

Comment:
The commented out part of this line should be removed if it isn't used any more.